www/js/shortcuts/shortcuts.js

```diff
@@ -0,0 +1,18 @@
+/* global Mousetrap */
+const shortcutsMap = require('./shortcuts-map.json');
+
+const shortcutFactory = (keys, actionName) => {
+  Mousetrap.bindGlobal(keys, () => {
```

Comment:
I'd like to swap `Mousetrap` out eventually.  Think this structure should be fine for now.